README.md

```diff
@@ -169,6 +169,28 @@ set SNOWFLAKE_AUTHENTICATOR=externalbrowser
 os.environ['SNOWFLAKE_AUTHENTICATOR'] = 'externalbrowser'
 ```
 
+4. Private Key based authentication
+```bash
+SNOWFLAKE_AUTHENTICATOR=
```

Comment:
It might be confusing to some folks about setting this to the empty string, I'd just leave a comment that this doesn't need to be set:
```suggestion
# no SNOWFLAKE_AUTHENTICATOR needed
```